deployment/config/mainnet/ecosystem.json

```diff
@@ -1,33 +1,34 @@
 {
+  "operatingChains": [2, 4, 5, 6, 23, 24, 30],
   "evm": {
     "networks": [
       {
         "name": "Ethereum",
         "externalId": 1,
         "chainId": 2,
         "network": "Mainnet",
-        "rpc": "https://rpc.ankr.com/eth"
+        "rpc": "https://eth-mainnet.g.alchemy.com/v2/n7D3C5-Th1KJHZgdyI0hjJR2XWBMzD9L"
```

Comment:
We should probably put another public RPC here. Bear in mind that this is a public repository :sweat_smile:

Reply:
fixed